src/configuration.ts

```diff
@@ -146,7 +146,7 @@ export interface IBaseConfiguration extends IMandatedConfiguration {
   /**
    * Timeouts for several operations (currently only source-maps)
    */
-  timeouts: Partial<SourceMapTimeouts>;
+  timeouts: Partial<SourceMapTimeouts> & HoverEvaluationTimeout;
```

Comment:
I think we can instead rename `SourceMapTimeouts` and add `hoverEvaluation` to it

Reply:
Thought about this already, but I wasn't sure, whether I should move the definition of `SourceMapTimeouts` then.

Reply:
The `SourceMapTimeouts` type is used in the `Sources` class. If the type is modified & renamed, anywhere in `Sources` where `SourceMapTimeouts` is currently used, the `hoverEvaluation` timeout needs to have been set, although `Sources` although `Sources` is completely unrelated to it and therefor never uses it.

Reply:
I pushed a commit where `SourceMapTimeouts` still is a separate interface, but the types are structured a little bit better.